static/app/views/dashboardsV2/widgetCard/chart.tsx

```diff
@@ -372,10 +386,10 @@ class WidgetCardChart extends React.Component<WidgetCardChartProps, State> {
     const chartOptions = {
       autoHeightResize,
       grid: {
-        left: 4,
-        right: 0,
+        left: 0,
+        right: 4,
         top: '40px',
-        bottom: 0,
+        bottom: showSlider ? '60px' : 0,
```

Comment:
Can we use that `SLIDER_HEIGHT` constant from `widgetViewerModal.tsx` here too to replace `'60px'`?

Reply:
👍  yup, i moved the const around since it seemed better to keep it in `chart.tsx` if we're exporting and sharing it.